packages/progress/src/index.vue

```diff
@@ -16,7 +16,7 @@
   >
     <div v-if="type === 'line'" class="el-progress-bar">
       <div class="el-progress-bar__outer" :style="{height: `${strokeWidth}px`}">
-        <div class="el-progress-bar__inner" :style="barStyle">
+        <div :class="`el-progress-bar__inner ${indeterminate ? 'el-progress-bar__inner__indeterminate' : ''}`" :style="barStyle">
```

Comment:
Why don't we use Array expressions here? Would've had better readability like this:
```vue
<div :class="[el-progress-bar__inner, indeterminate  ? el-progress-bar__inner--indeterminate : '']"
```
Also, according to our usage of BEM, you should use `--` instead of `__` to mark `indeterminate`.